packages/laconia-adapter-api/src/websocket.js

```diff
@@ -0,0 +1,12 @@
+const { res, parseWebsocket } = require("@laconia/event").apigateway;
```

Comment:
Do you think it should be parseWebSocket or parseWebsocket? (Notice the capital S)

Reply:
I guess this question will be applied throughout the change